pkg/skaffold/deploy/kubectl.go

```diff
@@ -65,6 +65,8 @@ spec:
 
 type KubectlDeployer struct {
 	*config.DeployConfig
+
+	buildTags   *BuildTags
```

Comment:
Should buildTags go on SkaffoldRunner instead?

Reply:
I can try but I'm not sure it makes sense